src/components/Project/Services/index.js

```diff
@@ -647,12 +646,13 @@ export default class Services extends React.Component {
         return {
           key: serviceSpec.id,
           value: serviceSpec.name,
-          tooltip: "Req: " + serviceSpec.cpuRequest + "mcpu | Lim: " + 
-                     serviceSpec.cpuLimit + "mcpu | Req: " + serviceSpec.memoryRequest + "mb | Lim: " + 
+          tooltip: "Req: " + serviceSpec.cpuRequest + "mcpu | Lim: " +
+                     serviceSpec.cpuLimit + "mcpu | Req: " + serviceSpec.memoryRequest + "mb | Lim: " +
```

Comment:
This all fits in the tooltip?

Reply:
Yes since I haven't modified this aside from removing whitespace